gui/preprocess_tool.py

```diff
@@ -318,8 +318,8 @@ def __init__(self, parent=None, stack=None):
             self.tb_fmt = 'tif'
             self.pad_bg_color = 'white'
 
-        self.stack_data_dir = os.path.join(thumbnail_data_dir, stack)
-        self.stack_data_dir_gordon = os.path.join(gordon_thumbnail_data_dir, stack)
+        self.stack_data_dir = data_dir + stack
```

Comment:
Change this line to my version. Create `thumbnail_data_dir` in `metadata`, then point it to `data_dir`.